doc/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -j auto -W --keep-going -w build_errors.txt -N -q
+SPHINXOPTS    = -j auto --keep-going -w build_errors.txt -N -q
```

Comment:
Did you mean to turn off -W?

Reply:
Yes, we are getting more than 25000 warnings thus it hangs so to avoid this we have removed `-W` option.